IngresseSDK/Model/Requests/EventRequests.swift

```diff
@@ -28,16 +28,32 @@ extension Request {
 
             public init() {}
         }
-        
-        public struct Search {
-            public var filters: [String: String] = [:]
-            public var orderBy: String = ""
-            public var from: String = ""
-            public var to: String = ""
-            public var offset: Int = 0
-            public var size: Int = 0
-            
-            public init() {}
+
+        public struct Search: Encodable {
+            public let orderBy: String
+            public let from: String
+            public let to: String
+            public let offset: Int
+            public let size: Int
+            public let title: String
+            public let description: String
```

Comment:
@PhillipiLino você pode deixar esses dois caras como `nil` assim, eles não serão "encodados".
E você não vai precisar fazer a validação se é vazio.

```suggestion
            public let title: String?
            public let description: String?
```